functions/dailyCC/dailyCC.js

```diff
@@ -0,0 +1,124 @@
+const axios = require("axios");
+const {slugs, baseUrl} = require('./constants')
+const {randomIndex, removeFromList} = require('../utilities')
+
+const getQuestion = async (slug)=>{
+	let url = baseUrl + '/' + slug
+	try{
+		let resp = await axios.get(url)
+		return resp.data
+	}
+	catch(err){
+		throw err
+	}
+}
+
+
+const getUsedSlugs = async (firestore)=>{
+	try{
+		let snapshot = await firestore.collection('dailyCC').doc('slugs').get();
+		if(snapshot.exists){
+			let slugs = snapshot.data()
+			return slugs.usedSlugs
+		}
+		else {
+			throw 'Snapshot does not existt'
+		}
+	}
+	catch(err){
+		throw err
+	}
+
+}
+
+const getPossibleSlugs = async({firestore, ignoreList=[]})=>{
+	try{
+		let snapshot = await firestore.collection('dailyCC').doc('slugs').get();
+
+		if(snapshot.exists){
+			let slugs = snapshot.data()
+			return removeFromList(slugs.allSlugs, ignoreList)
+		}
+		else {
+			throw 'Snapshot does not existt'
+		}
+	}
+	catch(err){
+		throw err
+	}
+
+}
+
+const addToUsedSlugs = async ({firestore, slug})=>{
+	try{
+		let usedSlugs = await getUsedSlugs(firestore)
+		usedSlugs.push(slug)
+
+		await firestore
+		.collection('dailyCC')
+		.doc('slugs')
+		.set({usedSlugs}, {merge: true})
+
+		return usedSlugs
+	}
+	catch(err){
+		throw err
+	}
+}
+
+//TODO error handling
+const handleDailyCC = async({channel, firestore})=>{
```

Comment:
Remove firestore since it'll be in its own config. Doesn't need to be an object that's passed here either.